2018/05/03 02:46:31 [warn] 1#1: conflicting server name "charset" on 0.0.0.0:80, ignored
2018/05/03 02:46:31 [warn] 1#1: conflicting server name "utf-8" on 0.0.0.0:80, ignored
2018/05/03 02:46:31 [warn] 1#1: conflicting server name "charset" on 0.0.0.0:80, ignored
2018/05/03 02:46:31 [warn] 1#1: conflicting server name "utf-8" on 0.0.0.0:80, ignored
2018/05/03 02:46:31 [warn] 1#1: conflicting server name "charset" on 0.0.0.0:80, ignored
2018/05/03 02:46:31 [warn] 1#1: conflicting server name "utf-8" on 0.0.0.0:80, ignored
2018/05/03 02:54:06 [error] 6#6: *16 FastCGI sent in stderr: "PHP message: PHP Warning:  Uncaught ErrorException: require_once(/var/www/html/eber-connector/routes/revel.php): failed to open stream: No such file or directory in /var/www/html/eber-connector/routes/web.php:23
Stack trace:
#0 /var/www/html/eber-connector/routes/web.php(23): Illuminate\Foundation\Bootstrap\HandleExceptions->handleError(2, 'require_once(/v...', '/var/www/html/e...', 23, Array)
#1 /var/www/html/eber-connector/routes/web.php(23): require_once()
#2 /var/www/html/eber-connector/vendor/laravel/framework/src/Illuminate/Routing/Router.php(394): require('/var/www/html/e...')
#3 /var/www/html/eber-connector/vendor/laravel/framework/src/Illuminate/Routing/Router.php(350): Illuminate\Routing\Router->loadRoutes('/var/www/html/e...')
#4 /var/www/html/eber-connector/vendor/laravel/framework/src/Illuminate/Routing/RouteRegistrar.php(105): Illuminate\Routing\Router->group(Array, '/var/www/html/e...')
#5 /var/www/html/eber-connector/app/Providers/RouteServiceProvider.php(56): Illuminate\Routing\RouteRegis...
PHP message: PHP Fatal error:  Illuminate\Routing\Router::main(): Failed opening required '/var/www/html/eber-connector/routes/revel.php' (include_path='.:/usr/local/lib/php') in /var/www/html/eber-connector/routes/web.php on line 23" while reading response header from upstream, client: 172.18.0.1, server: eber-connector.local, request: "GET /test HTTP/1.1", upstream: "fastcgi://172.18.0.4:9000", host: "eber-connector.local"
2018/05/03 09:05:10 [error] 6#6: *125 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Class 'App\Controller\VincentController\BaseController' not found in /var/www/html/eber-api/app/controllers/VincentController.php on line 11" while reading response header from upstream, client: 172.18.0.1, server: eber-api.local, request: "GET /vincent HTTP/1.1", upstream: "fastcgi://172.18.0.4:9000", host: "eber-api.local"
2018/05/03 09:05:27 [error] 6#6: *125 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Class 'App\Controller\BaseController\BaseController' not found in /var/www/html/eber-api/app/controllers/VincentController.php on line 12" while reading response header from upstream, client: 172.18.0.1, server: eber-api.local, request: "GET /vincent HTTP/1.1", upstream: "fastcgi://172.18.0.4:9000", host: "eber-api.local"
2018/05/03 09:05:30 [error] 6#6: *125 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Class 'App\Controller\BaseController' not found in /var/www/html/eber-api/app/controllers/VincentController.php on line 12" while reading response header from upstream, client: 172.18.0.1, server: eber-api.local, request: "GET /vincent HTTP/1.1", upstream: "fastcgi://172.18.0.4:9000", host: "eber-api.local"
2018/05/03 09:05:45 [error] 6#6: *125 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Class 'App\Controllers\BaseController' not found in /var/www/html/eber-api/app/controllers/VincentController.php on line 12" while reading response header from upstream, client: 172.18.0.1, server: eber-api.local, request: "GET /vincent HTTP/1.1", upstream: "fastcgi://172.18.0.4:9000", host: "eber-api.local"
2018/05/03 09:05:55 [error] 6#6: *125 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Class 'Yummypin\Controllers\BaseController' not found in /var/www/html/eber-api/app/controllers/VincentController.php on line 12" while reading response header from upstream, client: 172.18.0.1, server: eber-api.local, request: "GET /vincent HTTP/1.1", upstream: "fastcgi://172.18.0.4:9000", host: "eber-api.local"
2018/05/03 09:06:12 [error] 6#6: *125 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Class 'Yummypin\BaseController' not found in /var/www/html/eber-api/app/controllers/VincentController.php on line 12" while reading response header from upstream, client: 172.18.0.1, server: eber-api.local, request: "GET /vincent HTTP/1.1", upstream: "fastcgi://172.18.0.4:9000", host: "eber-api.local"
2018/05/03 09:14:57 [error] 6#6: *148 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Uncaught TypeError: Argument 1 passed to Illuminate\Exception\WhoopsDisplayer::display() must be an instance of Exception, instance of Error given, called in /var/www/html/eber-api/vendor/laravel/framework/src/Illuminate/Exception/Handler.php on line 279 and defined in /var/www/html/eber-api/vendor/laravel/framework/src/Illuminate/Exception/WhoopsDisplayer.php:43
Stack trace:
#0 /var/www/html/eber-api/vendor/laravel/framework/src/Illuminate/Exception/Handler.php(279): Illuminate\Exception\WhoopsDisplayer->display(Object(Error))
#1 /var/www/html/eber-api/vendor/laravel/framework/src/Illuminate/Exception/Handler.php(158): Illuminate\Exception\Handler->displayException(Object(Error))
#2 /var/www/html/eber-api/vendor/laravel/framework/src/Illuminate/Exception/Handler.php(169): Illuminate\Exception\Handler->handleException(Object(Error))
#3 [internal function]: Illuminate\Exception\Handler->handleUncaughtException(Object(Error))
#4 {main}
  thrown in /var/www/html/eber-api/vendor" while reading response header from upstream, client: 172.18.0.1, server: eber-api.local, request: "GET /vincent HTTP/1.1", upstream: "fastcgi://172.18.0.4:9000", host: "eber-api.local"
2018/05/03 09:15:20 [error] 6#6: *148 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Uncaught TypeError: Argument 1 passed to Illuminate\Exception\WhoopsDisplayer::display() must be an instance of Exception, instance of Error given, called in /var/www/html/eber-api/vendor/laravel/framework/src/Illuminate/Exception/Handler.php on line 279 and defined in /var/www/html/eber-api/vendor/laravel/framework/src/Illuminate/Exception/WhoopsDisplayer.php:43
Stack trace:
#0 /var/www/html/eber-api/vendor/laravel/framework/src/Illuminate/Exception/Handler.php(279): Illuminate\Exception\WhoopsDisplayer->display(Object(Error))
#1 /var/www/html/eber-api/vendor/laravel/framework/src/Illuminate/Exception/Handler.php(158): Illuminate\Exception\Handler->displayException(Object(Error))
#2 /var/www/html/eber-api/vendor/laravel/framework/src/Illuminate/Exception/Handler.php(169): Illuminate\Exception\Handler->handleException(Object(Error))
#3 [internal function]: Illuminate\Exception\Handler->handleUncaughtException(Object(Error))
#4 {main}
  thrown in /var/www/html/eber-api/vendor" while reading response header from upstream, client: 172.18.0.1, server: eber-api.local, request: "GET /vincent HTTP/1.1", upstream: "fastcgi://172.18.0.4:9000", host: "eber-api.local"
2018/05/03 09:15:45 [error] 6#6: *148 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Uncaught TypeError: Argument 1 passed to Illuminate\Exception\WhoopsDisplayer::display() must be an instance of Exception, instance of Error given, called in /var/www/html/eber-api/vendor/laravel/framework/src/Illuminate/Exception/Handler.php on line 279 and defined in /var/www/html/eber-api/vendor/laravel/framework/src/Illuminate/Exception/WhoopsDisplayer.php:43
Stack trace:
#0 /var/www/html/eber-api/vendor/laravel/framework/src/Illuminate/Exception/Handler.php(279): Illuminate\Exception\WhoopsDisplayer->display(Object(Error))
#1 /var/www/html/eber-api/vendor/laravel/framework/src/Illuminate/Exception/Handler.php(158): Illuminate\Exception\Handler->displayException(Object(Error))
#2 /var/www/html/eber-api/vendor/laravel/framework/src/Illuminate/Exception/Handler.php(169): Illuminate\Exception\Handler->handleException(Object(Error))
#3 [internal function]: Illuminate\Exception\Handler->handleUncaughtException(Object(Error))
#4 {main}
  thrown in /var/www/html/eber-api/vendor" while reading response header from upstream, client: 172.18.0.1, server: eber-api.local, request: "GET /vincent HTTP/1.1", upstream: "fastcgi://172.18.0.4:9000", host: "eber-api.local"
2018/05/03 09:16:00 [error] 6#6: *148 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Uncaught TypeError: Argument 1 passed to Illuminate\Exception\WhoopsDisplayer::display() must be an instance of Exception, instance of Error given, called in /var/www/html/eber-api/vendor/laravel/framework/src/Illuminate/Exception/Handler.php on line 279 and defined in /var/www/html/eber-api/vendor/laravel/framework/src/Illuminate/Exception/WhoopsDisplayer.php:43
Stack trace:
#0 /var/www/html/eber-api/vendor/laravel/framework/src/Illuminate/Exception/Handler.php(279): Illuminate\Exception\WhoopsDisplayer->display(Object(Error))
#1 /var/www/html/eber-api/vendor/laravel/framework/src/Illuminate/Exception/Handler.php(158): Illuminate\Exception\Handler->displayException(Object(Error))
#2 /var/www/html/eber-api/vendor/laravel/framework/src/Illuminate/Exception/Handler.php(169): Illuminate\Exception\Handler->handleException(Object(Error))
#3 [internal function]: Illuminate\Exception\Handler->handleUncaughtException(Object(Error))
#4 {main}
  thrown in /var/www/html/eber-api/vendor" while reading response header from upstream, client: 172.18.0.1, server: eber-api.local, request: "GET /vincent HTTP/1.1", upstream: "fastcgi://172.18.0.4:9000", host: "eber-api.local"
2018/05/03 09:17:41 [error] 6#6: *162 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Uncaught TypeError: Argument 1 passed to Illuminate\Exception\WhoopsDisplayer::display() must be an instance of Exception, instance of Error given, called in /var/www/html/eber-api/vendor/laravel/framework/src/Illuminate/Exception/Handler.php on line 279 and defined in /var/www/html/eber-api/vendor/laravel/framework/src/Illuminate/Exception/WhoopsDisplayer.php:43
Stack trace:
#0 /var/www/html/eber-api/vendor/laravel/framework/src/Illuminate/Exception/Handler.php(279): Illuminate\Exception\WhoopsDisplayer->display(Object(Error))
#1 /var/www/html/eber-api/vendor/laravel/framework/src/Illuminate/Exception/Handler.php(158): Illuminate\Exception\Handler->displayException(Object(Error))
#2 /var/www/html/eber-api/vendor/laravel/framework/src/Illuminate/Exception/Handler.php(169): Illuminate\Exception\Handler->handleException(Object(Error))
#3 [internal function]: Illuminate\Exception\Handler->handleUncaughtException(Object(Error))
#4 {main}
  thrown in /var/www/html/eber-api/vendor" while reading response header from upstream, client: 172.18.0.1, server: eber-api.local, request: "GET /vincent HTTP/1.1", upstream: "fastcgi://172.18.0.4:9000", host: "eber-api.local"
2018/05/03 09:17:56 [error] 6#6: *162 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Uncaught TypeError: Argument 1 passed to Illuminate\Exception\WhoopsDisplayer::display() must be an instance of Exception, instance of Error given, called in /var/www/html/eber-api/vendor/laravel/framework/src/Illuminate/Exception/Handler.php on line 279 and defined in /var/www/html/eber-api/vendor/laravel/framework/src/Illuminate/Exception/WhoopsDisplayer.php:43
Stack trace:
#0 /var/www/html/eber-api/vendor/laravel/framework/src/Illuminate/Exception/Handler.php(279): Illuminate\Exception\WhoopsDisplayer->display(Object(Error))
#1 /var/www/html/eber-api/vendor/laravel/framework/src/Illuminate/Exception/Handler.php(158): Illuminate\Exception\Handler->displayException(Object(Error))
#2 /var/www/html/eber-api/vendor/laravel/framework/src/Illuminate/Exception/Handler.php(169): Illuminate\Exception\Handler->handleException(Object(Error))
#3 [internal function]: Illuminate\Exception\Handler->handleUncaughtException(Object(Error))
#4 {main}
  thrown in /var/www/html/eber-api/vendor" while reading response header from upstream, client: 172.18.0.1, server: eber-api.local, request: "GET /vincent HTTP/1.1", upstream: "fastcgi://172.18.0.4:9000", host: "eber-api.local"
2018/05/03 09:19:37 [error] 6#6: *162 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Uncaught TypeError: Argument 1 passed to Illuminate\Exception\WhoopsDisplayer::display() must be an instance of Exception, instance of Error given, called in /var/www/html/eber-api/vendor/laravel/framework/src/Illuminate/Exception/Handler.php on line 279 and defined in /var/www/html/eber-api/vendor/laravel/framework/src/Illuminate/Exception/WhoopsDisplayer.php:43
Stack trace:
#0 /var/www/html/eber-api/vendor/laravel/framework/src/Illuminate/Exception/Handler.php(279): Illuminate\Exception\WhoopsDisplayer->display(Object(Error))
#1 /var/www/html/eber-api/vendor/laravel/framework/src/Illuminate/Exception/Handler.php(158): Illuminate\Exception\Handler->displayException(Object(Error))
#2 /var/www/html/eber-api/vendor/laravel/framework/src/Illuminate/Exception/Handler.php(169): Illuminate\Exception\Handler->handleException(Object(Error))
#3 [internal function]: Illuminate\Exception\Handler->handleUncaughtException(Object(Error))
#4 {main}
  thrown in /var/www/html/eber-api/vendor" while reading response header from upstream, client: 172.18.0.1, server: eber-api.local, request: "GET /vincent HTTP/1.1", upstream: "fastcgi://172.18.0.4:9000", host: "eber-api.local"
2018/05/03 09:19:59 [error] 6#6: *162 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Uncaught TypeError: Argument 1 passed to Illuminate\Exception\WhoopsDisplayer::display() must be an instance of Exception, instance of Error given, called in /var/www/html/eber-api/vendor/laravel/framework/src/Illuminate/Exception/Handler.php on line 279 and defined in /var/www/html/eber-api/vendor/laravel/framework/src/Illuminate/Exception/WhoopsDisplayer.php:43
Stack trace:
#0 /var/www/html/eber-api/vendor/laravel/framework/src/Illuminate/Exception/Handler.php(279): Illuminate\Exception\WhoopsDisplayer->display(Object(Error))
#1 /var/www/html/eber-api/vendor/laravel/framework/src/Illuminate/Exception/Handler.php(158): Illuminate\Exception\Handler->displayException(Object(Error))
#2 /var/www/html/eber-api/vendor/laravel/framework/src/Illuminate/Exception/Handler.php(169): Illuminate\Exception\Handler->handleException(Object(Error))
#3 [internal function]: Illuminate\Exception\Handler->handleUncaughtException(Object(Error))
#4 {main}
  thrown in /var/www/html/eber-api/vendor" while reading response header from upstream, client: 172.18.0.1, server: eber-api.local, request: "GET /vincent HTTP/1.1", upstream: "fastcgi://172.18.0.4:9000", host: "eber-api.local"
2018/05/03 09:20:09 [error] 6#6: *162 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Uncaught TypeError: Argument 1 passed to Illuminate\Exception\WhoopsDisplayer::display() must be an instance of Exception, instance of Error given, called in /var/www/html/eber-api/vendor/laravel/framework/src/Illuminate/Exception/Handler.php on line 279 and defined in /var/www/html/eber-api/vendor/laravel/framework/src/Illuminate/Exception/WhoopsDisplayer.php:43
Stack trace:
#0 /var/www/html/eber-api/vendor/laravel/framework/src/Illuminate/Exception/Handler.php(279): Illuminate\Exception\WhoopsDisplayer->display(Object(Error))
#1 /var/www/html/eber-api/vendor/laravel/framework/src/Illuminate/Exception/Handler.php(158): Illuminate\Exception\Handler->displayException(Object(Error))
#2 /var/www/html/eber-api/vendor/laravel/framework/src/Illuminate/Exception/Handler.php(169): Illuminate\Exception\Handler->handleException(Object(Error))
#3 [internal function]: Illuminate\Exception\Handler->handleUncaughtException(Object(Error))
#4 {main}
  thrown in /var/www/html/eber-api/vendor" while reading response header from upstream, client: 172.18.0.1, server: eber-api.local, request: "GET /vincent HTTP/1.1", upstream: "fastcgi://172.18.0.4:9000", host: "eber-api.local"
2018/05/03 09:26:09 [error] 6#6: *193 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Uncaught TypeError: Argument 1 passed to Illuminate\Exception\WhoopsDisplayer::display() must be an instance of Exception, instance of Error given, called in /var/www/html/eber-api/vendor/laravel/framework/src/Illuminate/Exception/Handler.php on line 279 and defined in /var/www/html/eber-api/vendor/laravel/framework/src/Illuminate/Exception/WhoopsDisplayer.php:43
Stack trace:
#0 /var/www/html/eber-api/vendor/laravel/framework/src/Illuminate/Exception/Handler.php(279): Illuminate\Exception\WhoopsDisplayer->display(Object(Error))
#1 /var/www/html/eber-api/vendor/laravel/framework/src/Illuminate/Exception/Handler.php(158): Illuminate\Exception\Handler->displayException(Object(Error))
#2 /var/www/html/eber-api/vendor/laravel/framework/src/Illuminate/Exception/Handler.php(169): Illuminate\Exception\Handler->handleException(Object(Error))
#3 [internal function]: Illuminate\Exception\Handler->handleUncaughtException(Object(Error))
#4 {main}
  thrown in /var/www/html/eber-api/vendor" while reading response header from upstream, client: 172.18.0.1, server: eber-api.local, request: "GET /vincent HTTP/1.1", upstream: "fastcgi://172.18.0.4:9000", host: "eber-api.local"
2018/05/03 09:26:31 [error] 6#6: *193 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Uncaught TypeError: Argument 1 passed to Illuminate\Exception\WhoopsDisplayer::display() must be an instance of Exception, instance of Error given, called in /var/www/html/eber-api/vendor/laravel/framework/src/Illuminate/Exception/Handler.php on line 279 and defined in /var/www/html/eber-api/vendor/laravel/framework/src/Illuminate/Exception/WhoopsDisplayer.php:43
Stack trace:
#0 /var/www/html/eber-api/vendor/laravel/framework/src/Illuminate/Exception/Handler.php(279): Illuminate\Exception\WhoopsDisplayer->display(Object(Error))
#1 /var/www/html/eber-api/vendor/laravel/framework/src/Illuminate/Exception/Handler.php(158): Illuminate\Exception\Handler->displayException(Object(Error))
#2 /var/www/html/eber-api/vendor/laravel/framework/src/Illuminate/Exception/Handler.php(169): Illuminate\Exception\Handler->handleException(Object(Error))
#3 [internal function]: Illuminate\Exception\Handler->handleUncaughtException(Object(Error))
#4 {main}
  thrown in /var/www/html/eber-api/vendor" while reading response header from upstream, client: 172.18.0.1, server: eber-api.local, request: "GET /vincent HTTP/1.1", upstream: "fastcgi://172.18.0.4:9000", host: "eber-api.local"
2018/05/04 01:17:53 [warn] 1#1: conflicting server name "charset" on 0.0.0.0:80, ignored
2018/05/04 01:17:53 [warn] 1#1: conflicting server name "utf-8" on 0.0.0.0:80, ignored
2018/05/04 01:17:53 [warn] 1#1: conflicting server name "charset" on 0.0.0.0:80, ignored
2018/05/04 01:17:53 [warn] 1#1: conflicting server name "utf-8" on 0.0.0.0:80, ignored
2018/05/04 01:17:53 [warn] 1#1: conflicting server name "charset" on 0.0.0.0:80, ignored
2018/05/04 01:17:53 [warn] 1#1: conflicting server name "utf-8" on 0.0.0.0:80, ignored
2018/05/04 01:32:23 [error] 6#6: *9 upstream timed out (110: Connection timed out) while reading response header from upstream, client: 172.18.0.1, server: eber-api.local, request: "GET /vincent HTTP/1.1", upstream: "fastcgi://172.18.0.4:9000", host: "eber-api.local"
2018/05/04 01:34:35 [warn] 1#1: conflicting server name "charset" on 0.0.0.0:80, ignored
2018/05/04 01:34:35 [warn] 1#1: conflicting server name "utf-8" on 0.0.0.0:80, ignored
2018/05/04 01:34:35 [warn] 1#1: conflicting server name "charset" on 0.0.0.0:80, ignored
2018/05/04 01:34:35 [warn] 1#1: conflicting server name "utf-8" on 0.0.0.0:80, ignored
2018/05/04 01:34:35 [warn] 1#1: conflicting server name "charset" on 0.0.0.0:80, ignored
2018/05/04 01:34:35 [warn] 1#1: conflicting server name "utf-8" on 0.0.0.0:80, ignored
2018/05/04 01:39:37 [error] 5#5: *14 upstream timed out (110: Connection timed out) while reading response header from upstream, client: 172.18.0.1, server: eber-api.local, request: "GET /vincent HTTP/1.1", upstream: "fastcgi://172.18.0.5:9000", host: "eber-api.local"
2018/05/04 01:45:55 [warn] 1#1: conflicting server name "charset" on 0.0.0.0:80, ignored
2018/05/04 01:45:55 [warn] 1#1: conflicting server name "utf-8" on 0.0.0.0:80, ignored
2018/05/04 01:45:55 [warn] 1#1: conflicting server name "charset" on 0.0.0.0:80, ignored
2018/05/04 01:45:55 [warn] 1#1: conflicting server name "utf-8" on 0.0.0.0:80, ignored
2018/05/04 01:45:55 [warn] 1#1: conflicting server name "charset" on 0.0.0.0:80, ignored
2018/05/04 01:45:55 [warn] 1#1: conflicting server name "utf-8" on 0.0.0.0:80, ignored
2018/05/04 01:47:03 [error] 6#6: *1 upstream timed out (110: Connection timed out) while reading response header from upstream, client: 172.18.0.1, server: eber-api.local, request: "GET /vincent HTTP/1.1", upstream: "fastcgi://172.18.0.5:9000", host: "eber-api.local"
2018/05/04 02:09:09 [error] 6#6: *5 upstream timed out (110: Connection timed out) while reading response header from upstream, client: 172.18.0.1, server: eber-api.local, request: "GET /vincent HTTP/1.1", upstream: "fastcgi://172.18.0.5:9000", host: "eber-api.local"
2018/05/04 02:24:48 [error] 6#6: *7 upstream timed out (110: Connection timed out) while reading response header from upstream, client: 172.18.0.1, server: eber-api.local, request: "GET /vincent HTTP/1.1", upstream: "fastcgi://172.18.0.5:9000", host: "eber-api.local"
2018/05/04 06:47:08 [error] 6#6: *20 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Uncaught TypeError: Argument 1 passed to Illuminate\Exception\WhoopsDisplayer::display() must be an instance of Exception, instance of Error given, called in /var/www/html/eber-api/vendor/laravel/framework/src/Illuminate/Exception/Handler.php on line 279 and defined in /var/www/html/eber-api/vendor/laravel/framework/src/Illuminate/Exception/WhoopsDisplayer.php:43
Stack trace:
#0 /var/www/html/eber-api/vendor/laravel/framework/src/Illuminate/Exception/Handler.php(279): Illuminate\Exception\WhoopsDisplayer->display(Object(Error))
#1 /var/www/html/eber-api/vendor/laravel/framework/src/Illuminate/Exception/Handler.php(158): Illuminate\Exception\Handler->displayException(Object(Error))
#2 /var/www/html/eber-api/vendor/laravel/framework/src/Illuminate/Exception/Handler.php(169): Illuminate\Exception\Handler->handleException(Object(Error))
#3 [internal function]: Illuminate\Exception\Handler->handleUncaughtException(Object(Error))
#4 {main}
  thrown in /var/www/html/eber-api/vendor" while reading response header from upstream, client: 172.18.0.1, server: eber-api.local, request: "GET /vincent HTTP/1.1", upstream: "fastcgi://172.18.0.5:9000", host: "eber-api.local"
2018/05/07 04:05:44 [warn] 1#1: conflicting server name "charset" on 0.0.0.0:80, ignored
2018/05/07 04:05:44 [warn] 1#1: conflicting server name "utf-8" on 0.0.0.0:80, ignored
2018/05/07 04:05:44 [warn] 1#1: conflicting server name "charset" on 0.0.0.0:80, ignored
2018/05/07 04:05:44 [warn] 1#1: conflicting server name "utf-8" on 0.0.0.0:80, ignored
2018/05/07 04:05:44 [warn] 1#1: conflicting server name "charset" on 0.0.0.0:80, ignored
2018/05/07 04:05:44 [warn] 1#1: conflicting server name "utf-8" on 0.0.0.0:80, ignored
